apps/generic-issuance-client/src/pages/pipeline/PipelineEditSection.tsx

```diff
@@ -88,14 +88,14 @@ export function PipelineEditSection({
       </Box>
 
       <FancyEditor
-        style={{ width: "800px", height: "350px" }}
+        style={{ width: "800px", height: "450px" }}
         language="json"
         value={editorValue}
         setValue={setEditorValue}
         readonly={ownedBySomeoneElse && !isAdminView}
```

Comment:
can we also change this to
```suggestion
        readonly={!isAdminView}
```

Reply:
actually, suggestion is slightly incorrect as we don't want it readOnly in the non-admin CSV case. but yeah i think this makes sense to batch with your other change

Reply:
fyi the readonly prop doesnt work properly rn anyway xd but sure lgtm